src/data_access/TheMealDB.java

```diff
@@ -27,7 +27,7 @@ public TheMealDB(RecipeFactory recipeFactory) {
         this.recipeFactory = recipeFactory;
     }
     @Override
-    public Recipe getRecipeById(String id) {
+    public Recipe searchRecipesById(String id) {
```

Comment:
Small nitpick here but this probably should be named "searchRecipeById" since we are only expecting 1 recipe back from the API. This can be refactored later as I'm assuming other files are already making use of this.

Reply:
Yeah, I noticed this too, but this name is derived from another interface (specifically the `SearchIdUserDataAccessInterface`).